pkg/envoy/bootstrap/types.go

```diff
@@ -8,20 +8,11 @@ import (
 
 var log = logger.New("envoy/bootstrap")
 
-// Config is the type used to represent the information needed to build the Envoy bootstrap config
-type Config struct {
-	// Admin port is the Envoy admin port
-	AdminPort uint32
-
-	// XDSClusterName is the name of the XDS cluster to connect to
-	XDSClusterName string
-
+// Builder is the type used to build the Envoy bootstrap config.
+type Builder struct {
```

Comment:
I think it would increase readability if we renamed `Builder` to `EnvoyBootstrapConfigBuilder`? This to me provides instant context and I understand what this struct and functions will do.

Reply:
My vote would be to keep `Builder`. The import "path" includes "envoy" and "bootstrap", so that context is contained there and duplicating it doesn't seem to provide any benefit.